_rules/SC4-1-2-role-attribute-has-valid-value.md

```diff
@@ -90,14 +90,30 @@ Element with multiple invalid `role` value.
 <input type="text" role="invalid role">
 ```
 
+#### Failed example 3
+
+Element with role attribute that is not empty (""), neither a valid `role` value.
+
+```html
+<input type="text" role=" ">
```

Comment:
I'd make this inapplicable, allowing "only whitespace" in here. I don't think we should be this strict about it.

Reply:
https://github.com/act-rules/act-rules.github.io/pull/430#discussion_r275361440

Reply:
See https://github.com/act-rules/act-rules.github.io/pull/430/files#r278572430